homeassistant/components/tado/sensor.py

```diff
@@ -183,6 +195,16 @@ def get_geofencing_mode(data) -> str:
     state_fn=lambda data: data.ac_power,
     attributes_fn=lambda data: {"time": data.ac_power_timestamp},
 )
+COMFORT_TEMPERATURE_ENTITY_DESCRIPTION = TadoSensorEntityDescription(
```

Comment:
I'm guessing this is a sensor for temperatures. I think we should add the native unit of measurement and the device class Temperature

Reply:
it's actually not, it is a sensor related to this red section in the app:
![image](https://github.com/home-assistant/core/assets/5701372/168c79b0-274d-46e9-adbe-5bb81e5f4328)
whith humidity comfort in horizontal and temperature comfort in vertical. Values are text only...

Reply:
Note that freshnees is the yellow part

Reply:
![image](https://github.com/home-assistant/core/assets/5701372/7898ad75-b4ad-4cd6-82b6-cd7f0f89068a)

Reply:
ooh right, what can the values be? We have a SensorDeviceClass for ENUMs, you can then add the possible values and them to the translation file, making it easier to make automations for. Example: https://github.com/home-assistant/core/blob/dev/homeassistant/components/ecoforest/sensor.py#L76

Reply:
i don't know the different values :'(
I can try to run some stress tests on the sensor 😛

Reply:
I mean this would be the best option, this way when you create automations you can see the possible values:

![image](https://github.com/home-assistant/core/assets/7083755/6594d192-213b-4927-bfa7-7cea254ed77b)